cri/stream/portforward/httpstream.go

```diff
@@ -111,13 +111,6 @@ func (h *httpStreamHandler) getStreamPair(requestID string) (*httpStreamPair, bo
 	return pair, true
 }
 
-// hasStreamPair returns a bool indicating if a stream pair for requestID exists.
-func (h *httpStreamHandler) hasStreamPair(requestID string) bool {
```

Comment:
Please help confirm whether this part could be removed. @Starnop 
Thanks.

Reply:
Look good to be removed.